File: src/main/java/io/github/robertomike/baradum/requests/OrderRequest.java
package io.github.robertomike.baradum.requests;

import io.github.robertomike.hefesto.enums.Sort;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class OrderRequest {
    private String field;
    private Sort sort = Sort.ASC;
}

File: baradum-core/src/main/kotlin/io/github/robertomike/baradum/core/requests/OrderRequest.kt
package io.github.robertomike.baradum.core.requests

import io.github.robertomike.baradum.core.enums.SortDirection

data class OrderRequest @JvmOverloads constructor(
    val field: String? = null, 
    val sort: SortDirection = SortDirection.ASC
)
